src/test/java/org/opensrp/web/rest/ClientFormResourceTest.java

```diff
@@ -413,7 +597,6 @@ public void testAddClientFormWhenGivenJSONWithMissingReferencesShouldReturn400()
     @Test
     public void testAddClientFormWhenGivenInvalidJSONShouldReturn400() throws Exception {
         String formIdentifier = "opd/reg.json";
-        String formVersion = "0.1.1";
```

Comment:
This were removed since my understanding was, we aren't making the form version optional but removing the ability for a user/dev to actually set one.  In that case we don't need to pass in a form version.